File: src/main/java/com/binance/api/client/BinanceApiIsolatedMarginRestClient.java
package com.binance.api.client;

public interface BinanceApiIsolatedMarginRestClient {
    // User stream endpoints

    /**
     * Start a new user data stream.
     *
     * @return a listen key that can be used with data streams
     */
    String startUserDataStream(String symbol);


    /**
     * PING a user data stream to prevent a time out.
     *
     * @param listenKey listen key that identifies a data stream
     */
    void keepAliveUserDataStream(String listenKey);
}

File: src/main/kotlin/com/binance/api/client/BinanceApiIsolatedMarginRestClient.kt
package com.binance.api.client

interface BinanceApiIsolatedMarginRestClient {
    // User stream endpoints
    /**
     * Start a new user data stream.
     *
     * @return a listen key that can be used with data streams
     */
    fun startUserDataStream(symbol: String): String

    /**
     * PING a user data stream to prevent a time out.
     *
     * @param listenKey listen key that identifies a data stream
     */
    fun keepAliveUserDataStream(listenKey: String)
}